data/campaigns/Northern_Rebirth/scenarios/11a_The_Eastern_Flank.cfg

```diff
@@ -390,7 +390,7 @@
             [then]
                 [message]
                     speaker=Tallin
-                    message= _ "Blast it! The coward has fled. Eryssa, your elves are good at moving quickly through the forests. Do you think they can overtake and pin him down while we deal with this fortress?"
+                    message= _ "Blast it! The coward has fled. Eryssa, your elves are good at moving quickly through the forests. Do you think they can overtake him while we deal with this fortress?"
```

Comment:
"pin him down" seems better to me, implying that they won't have enough troops to win, just enough to delay

Reply:
Overtake could mean the same thing.